web/src/lib/components/shared-components/status-box.svelte

```diff
@@ -36,18 +36,22 @@
     return 'bg-immich-primary dark:bg-immich-dark-primary';
   };
 
-  $: userInfo && onUpdate();
+  $: $user && onUpdate();
 
   onMount(async () => {
     await refresh();
   });
 
   const refresh = async () => {
     try {
-      [$serverInfoStore, userInfo] = await Promise.all([
-        api.serverInfoApi.getServerInfo().then(({ data }) => data),
-        api.userApi.getMyUserInfo().then(({ data }) => data),
-      ]);
+      if (!$serverInfoStore) {
+        const { data } = await api.serverInfoApi.getServerInfo();
+        $serverInfoStore = data;
+      }
+      if (!$user) {
+        const { data } = await api.userApi.getMyUserInfo();
+        $user = data;
+      }
```

Comment:
Shouldn't this be handled by the store?

Reply:
Yep, that AIP request is useless, correct me if I'm wrong, but you're always authenticated when you have the side-bar and that store is already handled by `auth.ts`